client/src/app/LandingPage.css

```diff
@@ -0,0 +1,49 @@
+.sec1-container{
+    display: flex;
+    width: 100%;
+    height: 800px;
+    justify-content: center;
+    align-items: center;
+}
+
+.sec1-box{
+    display: flex;
+    width: 90%;
+    height: 700px;
+    background-color: white;
+    border: 10px solid #E0E4F2;
+    box-shadow: #6d6d6d 0px 0px 10px;
+    border-radius: 20px; 
+    align-items: center;
+    justify-content: center;
+}
+
+.sec1-logo-box{
+    display: grid;
+    width: 400px;
+    height: 500px;
+    position: absolute;
+    text-align: center;
+    right: 15%;
+    top: 10%;
+}
+
+/* not relevant yet
+.sec1-text-wheel{
+    position: relative;
+    top: 50%;
+    right: 20%;
+    height: 200px;
+    width: 700px;
+    justify-self: center;
+} */
+
+.sec1-main-logo{
+    justify-self: center;
+}
+
+.sec1-Link{
+    position: relative;
```

Comment:
We don't need to change the position for this.

Reply:
positioning fixed